html.md

```diff
@@ -0,0 +1,38 @@
+# HTML
+
+
+## Intro
+Hypertext Markup Language (HTML) is the standard markup language for creating web pages and web applications. Web browsers receive HTML documents from a web server or from local storage and render the documents into multimedia web pages. HTML describes the structure of a web page semantically and originally included cues for the appearance of the document. HTML elements are the building blocks of HTML pages. With HTML constructs, images and other objects such as interactive forms may be embedded into the rendered page. [source](https://en.wikipedia.org/wiki/HTML)
+
+## Classification / acceptance questions
+Questions students could ask themeselves to evaluate their competence level.
+
+### Apprentice
+	* I know the difference between <head> and <body>
+	* I can change the tittle of website
+	* I can use basics body tags (<h1>, <a>, <p>)
+	* I know difference between order and unorder list
+	* I can use atributes like href and src
+	* I can make simple table
+
+### Intermediate 
+ **Prerequisite knowledge**: css Apprentice level
+	* I can add css stylesheet 
+	* I know difference between structures based on tables or on divs
+	* I can customize <meta> tag
+	* I can use advantages of HTML5
+	* I can attach video, audio or youtube links to my website
+### Advanced 
```

Comment:
In this point, you can mention about blocking and non-block assets loading methods.